### DISCUSSION TITLE
How to run OSP Suite ?

### DISCUSSION THREAD

**Discussion by egmnb** [Open-Systems-Pharmacology/Forum#1420] [How-Tos]
I am not only new for OSP but also Github. I have checked most of the things but could not find proper answer that would work for me.

Is there any detail documentation to how to run OSP ?

Accept my Apologise in advance.

**Comment by msevestre** [comment]
@egmnb 
Check the getting started 
https://docs.open-systems-pharmacology.org/open-systems-pharmacology-suite/getting-started

**Reply by egmnb** [reply]
I have checked but will do it again, dont know what I did miss there.

Thank you.

**Reply by msevestre** [reply]
@egmnb have you installed the software? then you need to start PK-Sim for instance.

**Reply by egmnb** [reply]
Yes, I have but did not work for me.

**Reply by msevestre** [reply]
if you want help, you'll need to be more explicit about the issue you have. What did not work, what did you do, what is the error you are seeing etc?

**Reply by egmnb** [reply]
It just needed a restart :).

Sorry for confusion.

Warm Regards
